automap/src/comm_layer/pmp.rs

```diff
@@ -1916,12 +1918,14 @@ mod tests {
                 remap_interval: Duration::from_secs(300)
             }
         );
-        TestLogHandler::new().exists_log_containing("INFO: timed_remap_test: Remapping port 6689");
+        TestLogHandler::new()
+            .exists_log_containing(&format!("INFO: {}: Remapping port 6689", test_name));
```

Comment:
This case and many other ones are good, I don't mind approving it, although I consider the Utkarshe's style in its full purity slightly better because he take advantage of the chance of inlining the variable to be printed inside the placeholder that sets where it should be printed.  

`INFO: {test_name}: Remapping port 6689"` 

It feels like a decently applied symbiosis with the new changes that have been introduced in Rust recently. 

Don't do it if you feel more comfortable this way.

Reply:
...okay...